kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/backstack/BackStackContainer.kt

```diff
@@ -134,14 +132,13 @@ open class BackStackContainer(
   }
 
   override fun onSaveInstanceState(): Parcelable {
-    showing?.let { viewStateCache.save(it) }
```

Comment:
No longer need the `save()` call, b/c Android built-in call is no longer busted.